app/content/views/comment.py

```diff
@@ -0,0 +1,61 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.response import Response
+
+from app.common.mixins import ActionMixin
+from app.common.permissions import BasicViewPermission
+from app.common.viewsets import BaseViewSet
+from app.content.models import Comment, User
+from app.content.serializers import (
+    ChildCommentSerializer,
+    CommentCreateSerializer,
+    CommentUpdateSerializer,
+)
+
+
+class CommentViewSet(BaseViewSet, ActionMixin):
+    permission_classes = [BasicViewPermission]
+    serializer_class = ChildCommentSerializer
+    queryset = Comment.objects.all()
+
+    def create(self, request):
+        data = request.data
+
+        user = get_object_or_404(User, user_id=request.id)
+
```

Comment:
her kan vi skrive user = request.user. Nå gjør vi et ekstra og unødvendig database kall for å hente en bruker